content/SCALE/SCALETutorials/DataProtection/TrueCloudTasks.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Managing TrueCloud Backup Tasks"
+description: "Provides instructions on how to set up a TrueCloud Backup Task and how to configure an Storj iX account to work with TrueNAS."
+weight: 5
+tags:
+- TrueCloud
+- cloud
+- backup
+keywords:
+- nas cloud storage
+- data protection
+- data backup and recovery
+aliases: 
+- cloudsynctasks/addstorjcloudsynctask
+---
+
+TrueNAS can send, receive, or synchronize data with the cloud storage provider Storj.
+TrueCloud Backup Tasks allow for single-time transfers or recurring transfers on a schedule.
+They are an effective method to back up data to a remote location.
+
+{{< hint type=important >}}
+To take advantage of the lower-cost benefits of the TrueCloud backup service, you must create your Storj iX account using the link provided on the **Add Cloud Credentials** screen.
+
+You must also create and authorize the storage buckets on Storj for use by SCALE.
+
+iXsystems is not responsible for any charges you incur using a third-party vendor with the TrueCloud Backup feature.
+{{< /hint >}}
+
+This procedure provides instructions to set up both Storj and SCALE.
+
+## TrueCloud Backup Task Requirements
+
+You must have all system storage (pool and datasets or zvols) configured and ready to receive or send data.
+
+## Creating a TrueCloud Backup Task
+
```

Comment:
I think we can modify this short procedure a bit to decrease some of the confusion. Line 41 describes what creating the cloud credential includes, but then we list them out. If you want to keep these then make them subtask indented under step 1 because these steps just repeat what you said step 1 includes.